content/api/rest-api.textile

```diff
@@ -322,8 +322,8 @@ Publish a message on a channel. Note that since the REST API is stateless, publi
 The request body contains message details and is an object of the form:
 
 bc[json]. {
-  name: <event name>,
   data: <message payload>,
+  name: <optional event name>,
```

Comment:
Hey @AndyTWF, 

Are `name` and `data` not both technically optional going by [RSL1e](https://sdk.ably.com/builds/ably/specification/main/features/#RSL1e) (and a quick test)?

Reply:
Yes technically, I shall update that!

Reply:
Done in f325ab04ac730e747cfec996d0bda82a80283db812bee4559b4cbe6a6dcaca9d